### PR TITLE
Aw/ansd interim fixes

### PR DESCRIPTION
**Pull request by awisema**

### CODE REVIEW

File: db/warehouse/migrate/20230911204506_add_column_to_ansd_events.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddColumnToAnsdEvents < ActiveRecord::Migration[6.1]
+  def change
+    add_column :ansd_events, :client_id, :string
```
Comment (eanders) [inline db/warehouse/migrate/20230911204506_add_column_to_ansd_events.rb:3]: why is this a string?  Should it be source_personal_id?

Comment (eanders) [reply db/warehouse/migrate/20230911204506_add_column_to_ansd_events.rb:3]: looks like it replaces an integer below.

File: drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/event.rb [hunk 2]
```diff
@@ -10,10 +10,6 @@ class Event < GrdaWarehouseBase
 
     belongs_to :enrollment
 
-    def client_id
-      enrollment.source_enrollment.client.id
-    end
```
Comment (eanders) [inline drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/event.rb:15]: So this was an `integer` but below has been replaced with `personal_id`

File: drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/report.rb [hunk 3]
```diff
@@ -133,11 +136,7 @@ def enrollment_scope
         preload(:enrollment, :client).
         entry.
         open_between(start_date: filter.start_date, end_date: filter.end_date)
-      filter.apply(scope, report_scope_source)
-    end
-
-    def report_scope_source
-      GrdaWarehouse::ServiceHistoryEnrollment.entry
+      scope.in_project(GrdaWarehouse::Hud::Project.where(id: filter.effective_project_ids))
```
Comment (eanders) [inline drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/report.rb:139]: removing `filter.apply` means you're only obeying date and project_ids if I'm reading this correctly.  Are there other options on the form?

Comment (eanders) [reply drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/report.rb:139]: Looks like funding sources at the very least.

File: drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/event.rb (hunk 2 above)
Comment (awisema) [reply drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/event.rb:15]: Yeah -- there is a bit of history here -- they all began life as warehouse IDs, and then it turned out they needed to match the sending system, and they wanted all of the events, not just the ones attached to the included enrollments, so the original decision to associate the events w/ their enrollments got removed, but this code didn't.

File: db/warehouse/migrate/20230911204506_add_column_to_ansd_events.rb (hunk 1 above)
Comment (awisema) [reply db/warehouse/migrate/20230911204506_add_column_to_ansd_events.rb:3]: Good point -- I'll rename the column.

**Pull request merged**